### ISSUE TITLE
ModuleNotFoundError for version 2.4.1

### ISSUE THREAD

**Issue by bradenyang** [rpomponio/neuroHarmonize#45]
Hi,

Just wanted to first say that this is a really great tool that you've developed!

I am currently running into an issue with installing version 2.4.1 of the package via pip. I have a conda environment set up, and when I install the package and try to import it in python, I get this error:

```
>>> import neuroHarmonize
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'neuroHarmonize'
```

Upon inspection of where the package was installed in my conda environment, I see a directory named "neuroHarmonize-2.4.1.dist-info", but no actual python module with the accompanying code. I also tried reinstalling a previous version of the package (2.3.0), and that seemed to work just fine. Any idea about what might be going on? Thanks in advance!

**Comment by aboccali** [comment]
Hi,
I have the same problem!

**Comment by clarkenj** [comment]
Following as I have the same problem!

**Comment by rpomponio** [comment]
Hello,

Thank you all for raising this to my attention. I believe it is a simple mistake on my end that I am working on now.

I will try to upload a patched version to the python packaging index.

Will keep this issue open until then.

**Comment by rpomponio** [comment]
I believe this is now fixed with version `2.4.2`, which is on the [packaging index](https://pypi.org/project/neuroHarmonize/2.4.2/).

I'll close this issue but feel free to re-open it if the problem persists.

FYI - as a workaround, installing version `2.4.1` with GitHub was successful.
